biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceUtils.java

```diff
@@ -552,4 +552,10 @@ public static VersionedClause toVersionClause(Resource resource, String mask) {
 		return new VersionedClause(identity, attribs);
 	}
 
+	static <T> T requireNonNull(T obj) {
```

Comment:
This doesn't have any direct relation to Resources, so maybe there is a better location for it?